From b47ed7ce82a6c96a061e2691d39b7d8ba2f027c4 Mon Sep 17 00:00:00 2001
From: Hans Yang <yhx@rock-chips.com>
Date: Mon, 9 Nov 2020 19:24:55 +0800
Subject: [PATCH] arm64: configs: rockchip_linux: Enable
 CONFIG_MMC_SDHCI_OF_DWCMSHC

DesignWare based SDHCI controller will be used for eMMC on some
Rockchip platforms such as RK3568.

Signed-off-by: Hans Yang <yhx@rock-chips.com>
Change-Id: Icfc6b06d92267114bac44b06db96c923befee499
---
 arch/arm64/configs/rockchip_linux_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/rockchip_linux_defconfig b/arch/arm64/configs/rockchip_linux_defconfig
index ef02a094798e..8a9ff38c5971 100644
--- a/arch/arm64/configs/rockchip_linux_defconfig
+++ b/arch/arm64/configs/rockchip_linux_defconfig
@@ -392,6 +392,7 @@ CONFIG_MMC_TEST=y
 CONFIG_MMC_SDHCI=y
 CONFIG_MMC_SDHCI_PLTFM=y
 CONFIG_MMC_SDHCI_OF_ARASAN=y
+CONFIG_MMC_SDHCI_OF_DWCMSHC=y
 CONFIG_MMC_DW=y
 CONFIG_MMC_DW_ROCKCHIP=y
 CONFIG_NEW_LEDS=y
-- 
2.35.3

